src/exercises/01-Query-Renderer/README.md

```diff
@@ -314,6 +314,52 @@ We're already grabbing the artistID from the route params. We need to pass it in
 
 _./Artist1QueryRenderer.tsx_
 
+The last step for relay to understand which variable you are passing is to pass the variable next to Artist1QueryRendererQuery like this:
+
+```typescript
+     query Artist1QueryRendererQuery(**$artistID: ID!**){ 
+        artist(**id: $artistID**){
+           name
+           birthYear
+         }
+     }
+```
```

Comment:
I think that this is covered in [the previous step of the instructions](https://github.com/artsy/relay-workshop/blob/e4c96159012fe68fa82930560ec17af69c203083/src/exercises/01-Query-Renderer/README.md?plain=1#L258-L280). Am I misunderstanding and there's a difference between them that I'm not catching?

And if they are the same step, and it was something you missed when going through, is there something we can do to make it harder to miss?